src/index.js

```diff
@@ -85,7 +85,7 @@ async function authenticate(login, password) {
   const respBody = resp.body
 
   if (
-    respBody.message.includes('Identifiant ou mot de passe erroné') &&
+    respBody.message.includes('ou mot de passe erroné') &&
```

Comment:
It won't work, because not only there's a mistake on the first word, there's also one on the last: `érroné` vs `erronné`. 
So I'd suggest this change: `respBody.message.includes('ou mot de passe') &&`

Reply:
https://github.com/konnectors/ensap/pull/277 thanks